src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -164,6 +164,7 @@ function newRepeatableElement(container, field_group, values) {
             if (values != null) {
                 new_field_group.find('input, select, textarea').each(function () {
                     if ($(this).data('repeatable-input-name')) {
+                        $(this).attr('data-value', values[$(this).data('repeatable-input-name')]);
```

Comment:
Hmm... is this really needed? I mean:
- you can tell that an input has been processed by repeatable by the fact that it has a ```data-repeatable-input-name``` attribute;
- if the above is true, you can get its value using ```.val()```;

It seems like a duplication - then the value would be present in two places, both as a ```value``` and as a ```data-value```, which makes it difficult to understand which is which, and SUPER easy to get them out of sync, causing other problems down the line.

Reply:
Unfortunatelly select2 does not accept a simple `select2.val(2)` unless there is a option with key 2. 

When loading in page we have no way of getting the `.val()` from element, it just returns null. 

That's why an attr with value is needed. 

About beeing out of sync, we might create and event `repeatable-finished-loading` and remove the `data-value` from the fields.

Reply:
Ugh. This is getting too complicated - there must be a better way.

How about if, inside the ```select2```s, we make sure the call that gets the _default_ entries (the empty AJAX call) also includes the IDs of existing items, so that the response also includes them too, so that we're 100% sure that we can do ```val(2)``` on it?

Reply:
@pxpm just gone through this one more time, and what we talked over Skype seems like the only reasonable way to proceed:
- only on ```select```
- only if the options are not already there
- add ```data-options```, so that they're then converted into actual options by the init function